itou/templates/layout/base.html

```diff
@@ -64,6 +64,10 @@
         <div class="global-messages-container">
             {% block global_messages %}
                 {% include "includes/demo_accounts.html" %}
+                <div class="alert alert-info alert-dismissible-once fade show" role="status" id="zendesk-banner">
```

Comment:
Lorsqu'on clique sur le bouton Fermer et qu'on recharge la page, le bandeau revient.
Je proposerais : 
- soit d'enlever le bouton Fermer
- soit de modifier le code comme suit

```suggestion
                <div class="alert alert-info alert-dismissible-once fade show d-none" role="status" id="zendesk-banner">
```

Reply:
En effet, j'ai venu que [même l'exemple ne fonctionne plus](https://zeroheight.com/85c89893b/p/0775de-alerts). Ajouter le `d-none` cache la bannière complètement (en tous cas dans mon test).
Je propose de mettre en prod en l'état et de refaire une PR plus tard (@hellodeloo) pour corriger le fait qu'elle ne disparaisse pas définitivement.

Reply:
Merci pour le retour !